domFoundations/events,storage,attributes/dateRenderer.js

```diff
@@ -0,0 +1,23 @@
+import { timeText } from "./initialPageRenderer.js";
+import { dateText } from "./initialPageRenderer.js";
+import { saveData } from "./storage.js";
+import { loadData } from "./storage.js";
+
+export const renderDate = () => {
+    const dt = new Date();
+    const padL = (nr, len = 2, chr = `0`) => `${nr}`.padStart(2, chr);
+    
+    const date = `${padL(dt.getDate())}-${
+        padL(dt.getMonth()+1)}-${
+        dt.getFullYear()}`
+    saveData('date', date);
+
+    const time = `${
+        padL(dt.getHours())}:${
+        padL(dt.getMinutes())}:${
+        padL(dt.getSeconds())}`
+    saveData('time', time);
+
+    dateText.innerHTML = loadData('date');
+    timeText.innerHTML = loadData('time');
```

Comment:
why are you loading data if you have it a few lines above?

Reply:
The data is generated and saved to the local storage then it is loaded from the storage. It allows to preserve the data and render it back after the page refresh / reopenning.